src/bucket/test/BucketTests.cpp

```diff
@@ -319,6 +328,71 @@ generateDifferentAccount(std::vector<LedgerEntry> const& others)
     }
 }
 
+TEST_CASE("merges proceed old-style despite newer shadows",
+          "[bucket][bucketmaxprotocol]")
+{
+    VirtualClock clock;
+    Config const& cfg = getTestConfig();
+    Application::pointer app = createTestApplication(clock, cfg);
+    auto& bm = app->getBucketManager();
+    auto vers = getAppLedgerVersion(app);
```

Comment:
This looks weird: won't the test stop working in version 13? I think vers should be initialized to `Bucket::FIRST_PROTOCOL_SHADOWS_REMOVED`

Reply:
fixed